src/types/GenericItem.ts

```diff
@@ -0,0 +1,31 @@
+import { ContentBlock } from './ContentBlock';
+import { DataProvider } from './DataProvider';
+import { SVA_Date } from './Date';
+import { MediaContent } from './MediaContent';
+import { WebUrl } from './WebUrl';
+
+// This type is not complete. Only the currently used fields have been added.
```

Comment:
could this comment better be flagged with `TODO:`, `HINT:`, `NOTE:` or something to be more catchy for our future selfs?

Reply:
added `HINT`